pubtools/_quay/push_docker.py

```diff
@@ -611,6 +611,7 @@ def run(self) -> None:
             )
         # Sign containers with signers that doesn't need have pushed containers
         # in destination registry
+        set_aws_kms_environment_variables(self.target_settings, "cosign_signer")
```

Comment:
Should this be also added before signing prepush index images? Otherwise it will be overwritten by SBOM signing variables